app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/manager/Fido2CredentialManagerImpl.kt

```diff
@@ -65,27 +58,23 @@ class Fido2CredentialManagerImpl(
         val origin = fido2CredentialRequest.origin
             ?: fido2CredentialRequest.callingAppInfo.getAppOrigin()
 
-        return vaultSdkSource.registerFido2Credential(
-            request = RegisterFido2CredentialRequest(
-                userId = userId,
-                origin = origin,
-                requestJson = """{"publicKey": ${fido2CredentialRequest.requestJson}}""",
-                clientData = clientData,
-                selectedCipherView = selectedCipherView,
-                isUserVerificationSupported = true,
-            ),
-            fido2CredentialStore = this,
-            // TODO: [PM-8137] Determine if user verification is supported
-            checkUser = { _, _ -> CheckUserResult(true, true) },
-            checkUserAndPickCredential = { _, _ -> CipherViewWrapper(selectedCipherView) },
-        )
+        return vaultSdkSource
+            .registerFido2Credential(
+                request = RegisterFido2CredentialRequest(
+                    userId = userId,
+                    origin = origin,
+                    requestJson = """{"publicKey": ${fido2CredentialRequest.requestJson}}""",
+                    clientData = clientData,
+                    selectedCipherView = selectedCipherView,
+                    isUserVerificationSupported = true,
```

Comment:
We're implicitly sending `true` here because UV is handled before engaging the SDK at this time. Proper implementation of the SDK user interface will be done in a separate, more focused, PR.